data_steward/test/unit_test/bq_utils_test.py

```diff
@@ -4,7 +4,11 @@
 import bq_utils
 import gcs_utils
 from google.appengine.ext import testbed
-from test_util import FAKE_HPO_ID, FIVE_PERSONS_PERSON_CSV, NYC_FIVE_PERSONS_PERSON_CSV, PITT_FIVE_PERSONS_PERSON_CSV
+from test_util import FAKE_HPO_ID, \
```

Comment:
Backslashes make me sad, I will reformat this to one-liner if that's ok with you...

Reply:
That's fine with me. Will follow that next time on.